object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,14 @@
+/* exported omit */
+function omit(source, key) {
+  var object = {};
+  var array = Object.keys(source);
+  for (var i = 0; i < key.length; i++) {
+    var word = key[i];
+    if (array.length === 0) {
```

Comment:
this should be outside the loop. There is no reason to loop through anything if you are given an empty object